docs/dcp2_system_design.rst

```diff
@@ -1199,6 +1224,82 @@ for a project with one non-stratified matrix.
    that file may have a more up-to-date list
 
 
+Describing CGMs as analysis files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The mechanism defined in `Describing CGMs as supplementary files`_ is
+deprecated. Instead of representing a contributor-generated matrix (CGM) as a
+stand-alone supplementary file, it is described more accurately as an
+``analysis_file`` entity that is an output of a ``process`` entity.
+
+Information about the provenance of the CGMs for a particular project may be
+limited. The DCP/2 may not be in possession of intermediate artifacts like the
+sequence data or any information about those artifacts because, for example,
+they are subject to access restrictions the DCP/2 does not yet have the
+capabilities to enforce. Even if the DCP/2 is in possession of intermediate
+artifacts and knows their provenance, their bearing on the contributed
+matrices may not have been supplied.
```

Comment:
optional: do we want to include any disclaimers about batch correction here? a question for a scientific stakeholder almost.

Reply:
I think so but I wouldn't know what to write. Do you want to suggest specific language? Or should we ping someone who knows?

Reply:
I believe @ekiernan recently drafted this text for the portal for the disclaimer:

> **Interpret with caution: Data is not corrected for batch effects nor normalized across projects

Reply:
https://github.com/HumanCellAtlas/dcp2/issues/18